.vscode/launch.json

```diff
@@ -32,7 +32,7 @@
 			"restart": true,
 			"name": "Attach to Extension Host",
 			"timeout": 30000,
-			"port": 5870,
+			"port": 45099,
```

Comment:
this file better to revert to avoid conflicts on rebase with upstream

Reply:
Agree!